### PR TITLE
Bumped version to 0.2.0

### PR DESCRIPTION
**Pull request by jsmartt**
### Description

Merge this once we're ready to do the release
### Issues Resolved

(none)
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
  - [x] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [x] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

**Review by tmiotto — APPROVED** [review]
Are we good to go?

**Comment by jsmartt** [comment]
Ideally I guess I'd like to see #66 and #68 merged in before the release because they are bug fixes instead of new features. But otherwise... yep!

File: examples/unmanaged_device.rb [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+  data(
+    model: 'Procurve 4200VL',
+    deviceType: 'Server'
+  )
+end
+
+oneview_unmanaged_device 'UnmanagedDevice1' do
+  client my_client
+  action :delete
+end
```
Comment (jsmartt) [inline examples/unmanaged_device.rb:29]: This "change" is just line-ending fixes. @anazard , you may want to check your Git config settings. I always set core.autocrlf to input.

Comment (anacampesan) [reply examples/unmanaged_device.rb:29]: will do! thanks @jsmartt

**Comment by jsmartt** [comment]
Cool! I'm ready to release if you guys are! :shipit:

**Pull request merged**